src/styles/styles.js

```diff
@@ -3533,6 +3590,60 @@ const styles = {
         textAlign: 'center',
     },
 
+    loginButtonRow: {
+        justifyContent: 'center',
+        width: '100%',
+        ...flex.flexRow,
```

Comment:
We aren't really being consistent here about spreading utility styles from elsewhere, so I'd recommend just changing this to `flexDirection: 'row'`